src/ooxml/java/org/apache/poi/xssf/streaming/StreamingSheetWriter.java

```
@@ -0,0 +1,87 @@
+==================================================================== */
+
+package org.apache.poi.xssf.streaming;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+
+import org.apache.poi.util.POILogFactory;
+import org.apache.poi.util.POILogger;
+
+public class StreamingSheetWriter extends SheetDataWriter {
+    private static final POILogger logger = POILogFactory.getLogger(StreamingSheetWriter.class);
+
+    public StreamingSheetWriter() throws IOException {
+    	throw new RuntimeException("StreamingSheetWriter requires OutputStream");
+	}
+
+    public StreamingSheetWriter(OutputStream out) throws IOException {
+    	super(createWriter(out));
+        logger.log(POILogger.DEBUG, "Preparing SSXSSF sheet writer");
+    }
+
+    @Override
+    public File createTempFile() throws IOException {
+    	throw new RuntimeException("Not supported with StreamingSheetWriter");
+    }
+    
+    @Override
+    public Writer createWriter(File fd) throws IOException {
+    	throw new RuntimeException("Not supported with StreamingSheetWriter");
+    }
+    
+    /**
+     * Create a writer for the sheet data.
+     * 
+     * @param  out the output stream to write to
+     */
+    protected static Writer createWriter(OutputStream out) throws IOException {
+        return new BufferedWriter(
+                new OutputStreamWriter(out, "UTF-8"));
+    }
+    
+    @Override
+    public void close() throws IOException {
+        _out.flush();
+    }
+    
+    @Override
+    public InputStream getWorksheetXMLInputStream() throws IOException {
+    	throw new RuntimeException("Not supported with StreamingSheetWriter");
+    }
+    
+    @Override
+    protected void finalize() throws Throwable {
+    	// super.finalize();
+    }
+    
+    
+    @Override
+    boolean dispose() throws IOException {
+    	try {
```

I don't think exception should be swallowed